e2e/run.sh

```
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPT_DIR=$(dirname $(readlink -f $0))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)